### ISSUE TITLE
prepare label error

### ISSUE THREAD

**Issue by lzhin** [RicherMans/Datadriven-GPVAD#23]
when I run python3 prepare_labels.py --pre ../pretrained_models/teacher1/model.pth csv_labels/balanced.csv softlabels/hdf5/balanced.h5 softlabels/csv/balanced.csv
I find a error ,the encoders/balanced.pth not exist. I download  https://github.com/RicherMans/GPV/blob/master/pretrained/gpv_f.pth  to encoders/balanced.pth according to the other issues. But have the error   encoder has not classes_

**Comment by RicherMans** [comment]
Hey,
Sorry that at that time I wrote some rather unreasonable code. 
As the code says, its only intended to be used for the teacher logits.
As the README says, run:

```bash
python3 prepare_labels.py --pre ../pretrained_models/teacher1/model.pth csv_labels/balanced.csv softlabels/hdf5/balanced.h5 softlabels/csv/balanced.csv
```

For other models as far as I remember just map 1 to the speech class.
Kind regards,
Heinrich
